<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>single_crystal_elastic_plot_widget</class>
 <widget class="QWidget" name="single_crystal_elastic_plot_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1965</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="options_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hBL_dataset_selector">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="tB_up">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>previous dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_down">
         <property name="text">
          <string>next dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hS_dataset_selector">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="lV_datalist">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="plot_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="plot_head_layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="tB_grid">
         <property name="toolTip">
          <string>Toggle grid lines</string>
         </property>
         <property name="text">
          <string>grid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_crystal_axes">
         <property name="toolTip">
          <string>Select the grid on unit cell principal axes</string>
         </property>
         <property name="text">
          <string>crystal_axes</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_projections">
         <property name="toolTip">
          <string>Toggle projections</string>
         </property>
         <property name="text">
          <string>projections</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_linestyle">
         <property name="toolTip">
          <string>Toggle edge color of triangles/quadliterals or change pointsize for scatter plot</string>
         </property>
         <property name="text">
          <string>linestyle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_save_data">
         <property name="toolTip">
          <string>Export displayed data to the directory specified for export</string>
         </property>
         <property name="text">
          <string>save_data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="plot_footer_layout">
       <item>
        <widget class="QLabel" name="l_x_min">
         <property name="text">
          <string>X&lt;sub&gt;min&lt;sub&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sB_x_min">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>-2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_x_max">
         <property name="text">
          <string>X&lt;sub&gt;max&lt;sub&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sB_x_max">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_y_min">
         <property name="text">
          <string>Y&lt;sub&gt;min&lt;sub&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sB_y_min">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>-2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_y_max">
         <property name="text">
          <string>Y&lt;sub&gt;max&lt;sub&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sB_y_max">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_z_min">
         <property name="text">
          <string>I&lt;sub&gt;min&lt;sub&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sB_z_min">
         <property name="minimum">
          <double>-999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_z_max">
         <property name="text">
          <string>I&lt;sub&gt;max&lt;sub&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sB_z_max">
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_log">
         <property name="text">
          <string>log</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combB_colormap">
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_invert_cb">
         <property name="text">
          <string>invert</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_fontsize">
         <property name="text">
          <string>FontSize:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sB_fontsize">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>11</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hS_plot_footer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
